tests/nested/manual/grade-signed-above-testkeys-boot/task.yaml

```diff
@@ -0,0 +1,131 @@
+summary: Test that snapd with testkeys on UC20 can boot a model with grade signed.
+
+systems: [ubuntu-20.04-64]
+
+environment:
+  # use tpm + secure boot to get full disk encryption, this is explicitly needed
+  # for grade: secured
+  NESTED_ENABLE_TPM: true
+  NESTED_ENABLE_SECURE_BOOT: true
+
+  # kvm doesn't work with tpm + secure boot on gce unfortunately
+  NESTED_ENABLE_KVM: false
```

Comment:
Once we landed #9380 this is no longer needed, yes?

Reply:
yes